apps/condo/domains/common/hooks/useValidations.ts

```diff
@@ -55,12 +57,18 @@ export const useValidations: () => ValidatorTypes = () => {
         }
     }
 
+    const numberValidator: Rule = {
+        pattern: /^\d+(\.?\d+)?$/g,
```

Comment:
What about ","....Seems like some meters uses it as separator....so, it's not user friendly :)